### ISSUE TITLE
After re-downloading the module package, the old version of the world will collapse after putting it into the new version

### ISSUE THREAD

**Issue by NoahsarkTaiwan** [AllTheMods/ATM-8#811]
The crash reported that a block entity tried to perform a "divide by zero" operation
But I have added projectE/Project Expansion and cyclic to it. maybe it has something to do with that?
crash reported:https://mclo.gs/z5Hzq9j

**Comment by Mitchell5200** [comment]
Looks to be a direct issue with project expansion, and pipez, either remove project expansion, or remove the highlighted pipe with NBT explorer
```
Name: pipez:universal_pipe // de.maxhenkel.pipez.blocks.tileentity.UniversalPipeTileEntity
--
12209 | Block: Block{pipez:universal_pipe}[down=true,east=true,has_data=true,north=true,south=false,up=false,waterlogged=false,west=true]
12210 | Block location: World: (448,113,-273), Section: (at 0,1,15 in 28,7,-18; chunk contains blocks 448,-64,-288 to 463,319,-273), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,-64,-512 to 511,319,-1)
12211 | Block: Block{pipez:universal_pipe}[down=true,east=true,has_data=true,north=true,south=false,up=false,waterlogged=false,west=true]
12212 | Block location: World: (448,113,-273), Section: (at 0,1,15 in 28,7,-18; chunk contains blocks 448,-64,-288 to 463,319,-273), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,-64,-512 to 511,319,-1)
```

**Comment by DonovanDMC** [comment]
This has been fixed in [1.0.6](https://github.com/DonovanDMC/ProjectExpansion/releases/tag/1.19.2-1.0.6)

**Comment by Mitchell5200** [comment]
> This has been fixed in [1.0.6](https://github.com/DonovanDMC/ProjectExpansion/releases/tag/1.19.2-1.0.6)

@NoahsarkTaiwan , as per the mod devs instructions, update project Expansion.
As those two mods are not in ATM8, and the dev has solved this issue, i am now closing this issue post.
